docs/csharp/how-to/parse-strings-using-split.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "How to: Parse Strings Using String.Split (C# Guide)"
+description: String.Split returns an array of strings split from a set of delimiters. It's an easy way to parse strings.
+ms.date: 01/03/2018
+ms.prod: .net
+ms.technology: 
+  - "devlang-csharp"
+ms.topic: "article"
+helpviewer_keywords: 
+  - "splitting strings [C#]"
+  - "Split method [C#]"
+  - "strings [C#], splitting"
+  - "parse strings"
+ms.assetid: 729c2923-4169-41c6-9c90-ef176c1e2953
+author: "BillWagner"
+ms.author: "wiwagn"
+ms.custom: mvc
+---
+# How to: Parse Strings Using String.Split (C# Guide)
+
+The <xref:System.String.Split%2A?displayProperty=nameWithType> method creates an
+array of substrings by splitting the input string based on one or more delimiters.
+
+It is often the easiest way to separate a string on word boundaries. It is also used
+to split strings on other specific characters or strings.
+
+The following code splits a common phrase into an array of strings for each word.
+Try it yourself by pressing the *Run* button.
+
+[!code-csharp-interactive[split strings on word boundaries](../../../samples/snippets/csharp/how-to/strings/ParseStringsUsingSplit.cs#1)]
+
+Every instance of one of the separator characters produces a value in the
+returned array. Consecutive separator characters produce the empty string
+as a value in the returned array.  You can see this in the following example:
+
+[!code-csharp-interactive[split strings with repeated separators](../../../samples/snippets/csharp/how-to/strings/ParseStringsUsingSplit.cs#2)]
+
+This behvior makes it easier for formats like comma separated values (CSV)
+files representing tabular data. Consecutive commas represent a blank column.
+
+You can pass an optional <xref:System.StringSplitOptions.RemoveEmptyEntries> parameter to
+exclude any empty strings in the output. For more complicated processing of the returned
+ecollection, you can use [LINQ](../programming-guide/concepts/linq/index.md) to manipulate
+the result sequence.    
+
+<xref:System.String.Split%2A?displayProperty=nameWithType> can use
+multiple separator characters. The following example uses spaces, commas, periods, colons, and tabs, all passed in an array containing these separating characters to <xref:System.String.Split%2A>.  Each word in the target string's sentence displays separately from the resulting array of strings.  
```

Comment:
comma after "separating characters"
I'm not quite sure what the last sentence means.